slaves/linux-cross/Dockerfile

```diff
@@ -101,8 +104,18 @@ RUN                                               \
   for f in `ls /x-tools/mipsel-unknown-linux-musl/bin/mipsel-unknown-linux-musl-*`; do     \
     g=`basename $f`;  \
     ln -vs $f /usr/bin/`echo $g | sed -e 's/-unknown//'`;    \
+  done &&                                        \
+  for f in `ls /x-tools/armv7-unknown-linux-gnueabihf/bin/armv7-unknown-linux-gnueabihf-*`; do     \
+    g=`basename $f`;  \
+    ln -vs $f /usr/bin/`echo $g | sed -e 's/-unknown//'`;    \
   done
 
+# Instruct rustbuild to use the armv7-linux-gnueabihf toolchain instead of the default
+# arm-linux-gnueabihf one
+ENV AR_armv7_unknown_linux_gnueabihf=armv7-linux-gnueabihf-ar \
+  CC_armv7_unknown_linux_gnueabihf=armv7-linux-gnueabihf-gcc \
+  CXX_armv7_unknown_linux_gnueabihf=armv7-linux-gnueabihf-g++
```

Comment:
> Perhaps this could go ahead and add environment variables for basically all the cross triples?

We don't really *need* to set all the env variable for all the cross triple because the default toolchains chosen by the gcc crate(?) match the toolchains installed in this docker image. Unless you want to be explicit about it. In any case, where can I get the full list of cross triples we are currently building binaries for?